lib/element.rb

```diff
@@ -118,13 +118,26 @@ def value
   end
 
   def click
-    Log.debug("Clicking on #{self}")
+    Log.debug("[GRIDIUM::Element] Clicking on #{self}")
+    click_retry = 2
     if element.enabled?
       ElementExtensions.highlight(self) if Gridium.config.highlight_verifications
       $verification_passes += 1
-      element.click
+      begin
+        element.click
+      rescue Exception => e
+        Log.warn("[GRIDIUM::Element] Click Exception retrying...")
+        sleep 0.15
+        click_retry -= 1
+        if click_retry > 0
```

Comment:
if we've got 2 click retries as stated on line 122, here we will only retry once. Looks like should be `if click_retry >= 0`